src/utils.js

```diff
@@ -0,0 +1,70 @@
+import parse from 'csv-parse';
+import 'isomorphic-fetch'
+
+const isBrowser = typeof window !== 'undefined';
+
+let fs
+if (!isBrowser) {
+  fs = require('fs');
+}
+
+class Utils {
```

Comment:
Your call here as more experienced js developer but just for me personally not obvious why to do not have module of functions instead of class with static methods. But pretty minor and subjective.